lib/plugins/expressionPlugin.js

```diff
@@ -42,10 +42,12 @@ function setJEXLTransforms(transformationMap) {
     jexlParser.setTransforms(transformationMap);
```

Comment:
An entry in CHANGES_NEXT_RELEASE about the changes in this PR should be included.

Reply:
Fixed 670ec99